x-pack/test/security_solution_api_integration/test_suites/security_solution_endpoint_api_int/services/index.ts

```diff
@@ -23,6 +25,8 @@ export const services = {
   endpointPolicyTestResources: EndpointPolicyTestResourcesProvider,
   endpointArtifactTestResources: EndpointArtifactsTestResources,
   rolesUsersProvider: RolesUsersProvider,
+  endpointDataStreamHelpers: SecuritySolutionEndpointDataStreamHelpers,
```

Comment:
Question: Is there any upside to adding those as services instead of regular imports in tests?

Reply:
From my perspective, it's mainly for maintenance. I tried to imported it straight away in the code, e.g.: in security_solution_endpoint folder, 

```
import {something} from '{somewhere}/security_solution_api_integration/test_suites/security_solution_endpoint_api_int/registry.ts'
```

It gave me a coupe of type check error: https://github.com/elastic/kibana/pull/187257#discussion_r1664402026
Therefore, I moved them to the services folder and follow the `getService` pattern to apply them in the code.